From 24040613a77cc47408b0831fae55ef1f5e64aeb9 Mon Sep 17 00:00:00 2001
From: Nikhil Benesch <nikhil.benesch@gmail.com>
Date: Mon, 20 Dec 2021 23:06:27 -0500
Subject: [PATCH] unifdef: remove conflicting strlcpy declaration for macOS

---
 scripts/unifdef.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/scripts/unifdef.c b/scripts/unifdef.c
index 30d459fb0709..4d8aa3a3be23 100644
--- a/scripts/unifdef.c
+++ b/scripts/unifdef.c
@@ -72,8 +72,6 @@ __FBSDID("$FreeBSD: /repoman/r/ncvs/src/usr.bin/unifdef/unifdef.c,v 1.20 2005/05
 #include <string.h>
 #include <unistd.h>

-size_t strlcpy(char *dst, const char *src, size_t siz);
-
 /* types of input lines: */
 typedef enum {
 	LT_TRUEI,		/* a true #if with ignore flag */
--
2.25.1

